lib/api/monitor.ex

```diff
@@ -0,0 +1,153 @@
+      %{janus: :list_sessions}
+      |> add_optional_param(:admin_secret, admin_secret)
+
+    with {:ok,
+          %{
+            "janus" => "success",
+            "sessions" => list
+          }} <-
+           Connection.call(connection, message) do
+      {:ok, list}
+    end
+    |> API.handle_api_error()
+  end
+
+  @doc """
+  Requests list of all handles currently active within given session.
+
+  ## Arguments
+  * `connection` - valid connection with Monitor API.
+  * `session_id` - targeted session's id.
+  * `admin_secret` - secret added to API request if authorization is required.
+  """
+  @spec list_handles(GenServer.server(), Session.session_id_t(), binary | nil) ::
+          {:error, atom} | {:ok, list(non_neg_integer())}
+  def list_handles(connection, session_id, admin_secret \\ nil) do
+    message =
+      %{
+        janus: :list_handles,
+        session_id: session_id
+      }
+      |> add_optional_param(:admin_secret, admin_secret)
+
+    with {:ok,
+          %{
+            "janus" => "success",
+            "handles" => list
+          }} <-
+           Connection.call(connection, message) do
+      {:ok, list}
+    end
+    |> API.handle_api_error()
+  end
+
+  @doc """
+  Requests information about specific handle.
+
+  ## Arguments
+  * `connection` - valid connection with Monitor API.
+  * `session_id` - session id of targeted handler.
+  * `handle_id` - targeted handler's id.
+  * `admin_secret` - secret added to API request if authorization is required.
+  """
+  @spec handle_info(
+          GenServer.server(),
+          Session.session_id_t(),
+          Session.plugin_handle_id(),
+          binary | nil
+        ) ::
+          {:error, :atom} | {:ok, handle_info}
+  def handle_info(connection, session_id, handle_id, admin_secret \\ nil) do
+    message =
+      %{
+        janus: :handle_info,
+        session_id: session_id,
+        handle_id: handle_id
+      }
+      |> add_optional_param(:admin_secret, admin_secret)
+
+    with {:ok,
+          %{
+            "janus" => "success",
+            "info" => info
+          }} <-
+           Connection.call(connection, message) do
+      accepted_keys = [
+        "session_id",
+        "session_last_activity",
+        "session_transport",
+        "handle_id",
+        "opaque_id",
+        "loop-running",
+        "created",
+        "current_time",
+        "plugin",
+        "plugin_specific",
+        "flags",
+        "agent-created",
+        "ice-mode",
+        "ice-role",
+        "sdps",
+        "queued-packets",
+        "streams"
+      ]
+
+      {:ok, take_string_keys_turn_to_atom(info, accepted_keys)}
+    end
+    |> API.handle_api_error()
+  end
+
+  # This would be beneficial in future PRs
+  defp add_optional_param(map, key, value)
+  defp add_optional_param(map, _key, nil), do: map
+  defp add_optional_param(map, key, value), do: Map.put(map, key, value)
+
+  defp take_string_keys_turn_to_atom(map, accepted_keys) do
```

Comment:
```suggestion
  defp take_accepted_keys_as_atoms(map, accepted_keys) do
```
I would also change accepted keys into module attribute and not pass it as argument since `String.to_atom` is rather dangerous and we need to be sure it won't be called with arbitrary list of strings